leshan-integration-tests/credentials/server.csr

```diff
@@ -1,9 +1,9 @@
------BEGIN NEW CERTIFICATE REQUEST-----
+-----BEGIN NEW CERTIFICATE REQUEST-----

```

Comment:
It is actually from @sbernard31 and may be because of a rebase/building it locally once.